lib/tasks/install.rake

```diff
@@ -26,6 +26,7 @@ module Packaging
     DESTDIR = ENV["DESTDIR"] || "/"
     YAST_DIR = DESTDIR + "/usr/share/YaST2/"
     YAST_LIB_DIR = DESTDIR + "/usr/lib/YaST2/"
+    YAST_ICON_BASE_DIR = DESTDIR + "/usr/share/icons/hicolor"
```

Comment:
I'm just not sure if we should rather use `/usr/share/YaST2/theme/current/icons` here...

In openSUSE `/usr/share/YaST2/theme/current/icons` is a symlink pointing to `/usr/share/icons/hicolor`. But in SLE it is a regular directory. Which means the icon in `/usr/share/icons/hicolor` probably won't be visible in SLE.

See https://github.com/yast/yast-yast2/blob/5762181d62762816a73fc040362c1efb5d97deed/library/general/src/modules/Directory.rb#L91

@jreidinger ?

Reply:
yep, symlink should be used, so `/usr/share/YaST2/theme/current/icons` is correct.